charts/terminal/values.yaml

```diff
@@ -56,8 +56,18 @@ global:
             bindAddress: ""
             port: 8081
           metrics:
-            bindAddress: 127.0.0.1
-            port: 8080
+            bindAddress: ""
+            port: 8443
+            tlsSecretName: # must contain 'tls.crt', 'tls.key' keys
+            tls:
```

Comment:
Shouldn't the invalid default values be a comment otherwise these default values are used and no required error is returned by helm.
```
# caBundle: |
#   -----BEGIN CERTIFICATE-----
#   Li4u
#   -----END CERTIFICATE-----
```

Reply:
✅